pubspec.yaml

```
@@ -6,7 +6,7 @@ repository: https://github.com/treeder/jsqr_flutter
 issue_tracker: https://github.com/treeder/jsqr_flutter/issues
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0-29.10.beta <3.0.0'
   flutter: ">=1.20.0 <2.0.0"
```

Should let this upgrade to flutter 2 as well.

```suggestion
  flutter: ">=1.20.0 <3.0.0"
```

---

thanks, i've remove the maximum constraint

---

> Thanks again

you're welcome.
will you update versioning of this package in pub.dev ?

---

Done. 0.1.1

---

thankyou.